src/Tribe/Views/V2/Views/Widgets/Widget_View.php

```diff
@@ -174,7 +174,7 @@ public function get_compatibility_classes() {
 		 * @var Theme_Compatibility $theme_compatibility
 		 */
 		$theme_compatibility = tribe( Theme_Compatibility::class );
-		$classes = $theme_compatibility->get_container_classes();
+		$classes = $theme_compatibility::get_container_classes();
```

Comment:
maybe just call the static method from the class directly rather than fetching the instance?